redfish-core/lib/log_services.hpp

```diff
@@ -1599,25 +1620,47 @@ inline void requestRoutesDBusEventLogEntryCollection(App& app)
                                     }
                                 }
                             }
+                            else if (interfaceMap.first ==
+                                     "org.open_power.Logging.PEL.Entry")
+                            {
+                                for (auto& propertyMap : interfaceMap.second)
```

Comment:
const auto& :  make it const

Reply:
I may need to do it for the entire file. I just followed the same convention.